modules/openapi-generator/src/main/resources/python/asyncio/rest.mustache

```diff
@@ -68,6 +71,20 @@ class RESTClientObject:
             trust_env=True
         )
 
+        self.retries = configuration.retries
```

Comment:
I think you can just skip the `self.` part here, as you are now only using this `retries` attribute in the same function. No need to keep this value as an attribute of the class.